.github/ISSUE_TEMPLATE/topic_submission.md

```diff
@@ -18,4 +18,12 @@ ASAP? next month? Whenever?
 Any information you'd like to provide about yourself, the presenter. This may including name, github username, organization, or project affiliations, etc. 
 
 **Additional Info**
-If applicable, please provide links to any relevant blogs, videos, git repos, etc relating to the suggested presentation. 
\ No newline at end of file
+If applicable, please provide links to any relevant blogs, videos, git repos, etc relating to the suggested presentation. 
+
+**Actionable Items**
```

Comment:
For an external contributor trying to propose their topic these Action Items may not be relevant. For example they might contribute their approved for finalized topic to the agenda document however the rest of the items seem like the organizers' responsibilities.

Should we state that here? Maybe like
```suggestion
**Actionable Items** (Following actions would be carried out by the Organizers of the Operate First Data Science Community Meetup)
```

Reply:
Sounds good :+1: